.github/workflows/testnet-labels.yml

```diff
@@ -0,0 +1,18 @@
+name: Tested on Testnet
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
```

Comment:
what does `synchronise` do ?

Reply:
I don't understand it much but you need it for label stuff to work 100% of the time haha

Reply:
"In the context of GitHub Actions, the synchronize event type for pull_request triggers a workflow whenever a pull request’s head branch is updated. This usually happens when new commits are pushed to the branch from which the pull request was created." :)